pkg/param/pagination/pagination.go

```diff
@@ -1,30 +1,33 @@
 package pagination
 
+// GetPage - pagination
```

Comment:
Let's add clarifying comments regarding how this structure is used in requests and in responses.

Reply:
Sure .